.drone-1.0.yml

```
@@ -3,8 +3,12 @@ kind: pipeline
 name: default
 type: kubernetes
 steps:
+  - name: generate-package-lock
+    image: node:18
+    commands:
+      - npm ci --package-lock-only
```

`npm ci --package-lock-only` still errors because there's no lockfile. This would need to be:
```suggestion
  - name: generate-package-lock
    image: node:18
    commands:
      - npm install --package-lock-only
```

At which point this is redundant and we might as well update the install step to run `npm install` instead of `npm ci`. The downside of either is we lose the ability to do repeatable builds - there is no longer a guarantee that a tested and reviewed PR uses the same dependency versions when merged.